traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -1237,6 +1248,22 @@ func getMidServers(edgeIDs []int, servers map[int]tc.ServerV40, dsID int, cdnID
 		FROM deliveryservice d
 		WHERE d.id = :ds_id) IS NULL
 	`
+		query = selectQuery + `
+	WHERE (SELECT ARRAY_AGG(ssc.server_capability) FROM server_server_capability ssc WHERE ssc."server" = ANY(
+	` + q + `)) @> (SELECT ARRAY_AGG(drc.required_capability) FROM deliveryservices_required_capability drc WHERE drc.deliveryservice_id = :ds_id)`
```

Comment:
Can this query be formatted nicer so we can tell what's going on here? Commenting the query would not hurt, either.